### ISSUE TITLE
Tests aren't sorted alphabetically in the list

### ISSUE THREAD

**Issue by bitwes** [bitwes/Gut#111]
The list of tests should be alphabetical.  The entire list could be alphabetical or it would be ok for them to be alphabetical per directory with the directories listed in the order they are defined in either the scene or the .gutconfig.json file.  The whole list is preferable but it isn't worth a ton of work.

**Comment by bitwes** [comment]
This seems to work, but haven't vetted it fully 

`gut.gd`
```
unc add_directory(path, prefix=_file_prefix, suffix=_file_extension):
	var d = Directory.new()
	# check for '' b/c the calls to addin the exported directories 1-6 will pass
	# '' if the field has not been populated.  This will cause res:// to be
	# processed which will include all files if include_subdirectories is true.
	if(path == '' or !d.dir_exists(path)):
		if(path != ''):
			_lgr.error(str('The path [', path, '] does not exist.'))
		return
	d.open(path)
	# true parameter tells list_dir_begin not to include "." and ".." diretories.
	d.list_dir_begin(true)

	# Traversing a directory is kinda odd.  You have to start the process of listing
	# the contents of a directory with list_dir_begin then use get_next until it
	# returns an empty string.  Then I guess you should end it.
	var fs_item = d.get_next()
	var full_path = ''
	var files = []
	var dirs = []
	
	while(fs_item != ''):
		full_path = path.plus_file(fs_item)

		#file_exists returns fasle for directories
		if(d.file_exists(full_path)):
			if(fs_item.begins_with(prefix) and fs_item.ends_with(suffix)):
				files.append(full_path)
		elif(get_include_subdirectories() and d.dir_exists(full_path)):
			dirs.append(full_path)

		fs_item = d.get_next()
	d.list_dir_end()

	files.sort()
	for i in range(files.size()):
		add_script(files[i])

	dirs.sort()
	for i in range(dirs.size()):
		add_directory(dirs[i], prefix, suffix)
```

**Comment by bitwes** [comment]
Fixed in 6.8.0
